Deployement_Script.sql

```diff
@@ -29,26 +29,110 @@ SET time_zone = "+00:00";
 --
 
 CREATE TABLE `Customer` (
-  `Cid` int(11) NOT NULL Primary Key Auto_Increment,
-  `Email` VARCHAR(50) NOT NULL,
+  `Cid` int(11) NOT NULL AUTO_INCREMENT,

```

Comment:
only issue is that this need to be primary key still

Reply:
It is a primary key.  See line 36.